tests/legacy-cli/e2e/tests/update/update-9.ts

```diff
@@ -1,30 +1,26 @@
 import { createProjectFromAsset } from '../../utils/assets';
 import { expectFileMatchToExist } from '../../utils/fs';
-import { installPackage, installWorkspacePackages, setRegistry } from '../../utils/packages';
+import { installWorkspacePackages, setRegistry } from '../../utils/packages';
 import { ng, noSilentNg } from '../../utils/process';
 import { isPrereleaseCli, useCIChrome, useCIDefaults } from '../../utils/project';
 
 export default async function () {
   // We need to use the public registry because in the local NPM server we don't have
   // older versions @angular/cli packages which would cause `npm install` during `ng update` to fail.
   try {
-    await createProjectFromAsset('8.0-project', true, true);
+    await createProjectFromAsset('9.0-project', true, true);
 
     await setRegistry(false);
     await installWorkspacePackages();
 
-    // Update Angular to 9
-    await installPackage('@angular/cli@8');
-    const { stdout } = await ng('update', '@angular/cli@9.x', '@angular/core@9.x');
```

Comment:
Could we keep this check to make sure migrations are actually running?  Can just move it down to the 10 execution.

Reply:
done